tests/test_fork_point.py

```diff
@@ -0,0 +1,45 @@
+from .mockers import (GitRepositorySandbox, assert_command, launch_command)
+
+import os
+
+
+class TestForkPoint:
+
+    def setup_method(self) -> None:
+
+        self.repo_sandbox = GitRepositorySandbox()
+
+        (
+            self.repo_sandbox
+            # Create the remote and sandbox repos, chdir into sandbox repo
+            .new_repo(self.repo_sandbox.remote_path, "--bare")
+            .new_repo(self.repo_sandbox.local_path)
+            .execute(f"git remote add origin {self.repo_sandbox.remote_path}")
+            .execute('git config user.email "tester@test.com"')
+            .execute('git config user.name "Tester Test"')
+        )
+
+    def test_fork_point(self) -> None:
+        """
+        Verify behaviour of a 'git machete fork-point' command.
+        """
+        fixed_committer_and_author_date = 'Mon 20 Aug 2018 20:19:19 +0200'
+        os.environ['GIT_COMMITTER_DATE'] = fixed_committer_and_author_date
```

Comment:
Hmm for the sake of simplicity, how about the following:

1. Add `os.environ['GIT_AUTHOR_DATE'] = fixed_committer_and_author_date`
2. Remove both `.execute(f'git commit --amend...')` calls

My hypothesis is that you'll still get reproducible results (AFAICS you'll even get the exact same commit hashes as now)

Reply:
Yes, it works, i actually tested it firstly and it didn't work but the fault was on the difference in tree hash, forgot to come back to it